pokecli.py

```diff
@@ -208,6 +208,17 @@ def init_config():
         type=int,
         default=50
     )
+
+    add_config(
+        parser,
+        load,
+        short_flag="-n",
+        long_flag="--navigator",
+        help="Set the navigator to be used(DEFAULT spiral)",
+        type=str,
+        default=None
```

Comment:
Add `default='spiral'` and will be able to transform the `if` in `bot.start` from:

```python
if self.config.navigator == None or self.config.navigator == 'spiral'
``` 

To 

```python
self.config.navigator == 'spiral'
```